packages/components/src/internal/components/forms/detail/DetailDisplay.spec.tsx

```diff
@@ -1,3 +1,4 @@
+import * as console from 'console';
```

Comment:
Remove this import.

Reply:
Ugh,  I don't know why intellij keeps doing this to me.